packages/cozy-konnector-libs/src/libs/saveFiles.js

```diff
@@ -373,19 +375,19 @@ async function getFileFromMetaData(
 async function getFileFromPath(entry, options) {
   try {
     log('debug', `Checking existence of ${getFilePath({ entry, options })}`)
-    const result = await cozy.files.statByPath(getFilePath({ entry, options }))
-    return result
+    const result = await client.collection(FILES).statByPath(getFilePath({ entry, options }))
```

Comment:
note to myself: provide a SQL fro client to get from path instead of using the collection / statByPath